### DISCUSSION TITLE
Make selected field type tool persist

### DISCUSSION THREAD

**Discussion by exo-pla-net** [docusealco/docuseal#181] [Ideas]
Currently, the Signature tool swaps to the text tool, after you make a single signature.

Thus, after you make a signature field, when you try to make another signature, you instead make a text field.

This is unintuitive. It should instead keep making new signature fields.

**Comment by omohokcoj** [comment]
@exo-pla-net text field is always drawn by default - since it's the most used field type.
A draw text field can be changed to a signature or date field via the types dropdown before the field name.
I'm not sure if automatically switching a new field type to the previous field type might be more intuitive.
Converting it into the discussion so anyone can provide their suggestions regarding this.
